java-runtime-builder-app/src/test/java/com/google/cloud/runtimes/builder/buildsteps/PrebuiltRuntimeImageBuildStepTest.java

```diff
@@ -167,20 +167,6 @@ public void testUnrecognizedArtifact() throws IOException, BuildStepException {
     prebuiltRuntimeImageBuildStep.run(mockContext);
   }
 
-  @Test
-  public void testCompatArtifact() throws IOException, BuildStepException {
-    Path workspace = new TestWorkspaceBuilder()
-        .file("WEB-INF/appengine-web.xml").build()
-        .file("WEB-INF/web.xml").build()
-        .build();
-    BuildContext buildContext = new BuildContext(new AppYaml(), workspace, false);
-    prebuiltRuntimeImageBuildStep.run(buildContext);
-
-    String dockerfile = buildContext.getDockerfile().toString();
-    assertTrue(dockerfile.startsWith("FROM " + compatImageName + "\n"));
```

Comment:
I think we still need a test that verifies that the compat image is selected when `appengine-web.xml` is present.

Reply:
Ah , actually didn't know that was a rule! Previously having WEB-INF exist meant exploded-war, which automatically meant compat, but now I will add separate logic for that and a test.